hs_file_types/views.py

```diff
@@ -495,6 +515,45 @@ def update_dataset_name(request, hs_file_type, file_type_id, **kwargs):
 
 
 @login_required
+def update_refts_abstract(request, file_type_id, **kwargs):
+    """updates the abstract for ref time series specified logical file object
+    """
+
+    logical_file, json_response = _get_logical_file('RefTimeseriesLogicalFile', file_type_id)
+    if json_response is not None:
+        return json_response
+
+    resource_id = logical_file.resource.short_id
+    resource, authorized, _ = authorize(request, resource_id,
+                                        needed_permission=ACTION_TO_AUTHORIZE.EDIT_RESOURCE,
+                                        raises_exception=False)
+    if not authorized:
+        ajax_response_data = {'status': 'error', 'logical_file_type': logical_file.type_name(),
+                              'element_name': 'abstract', 'message': "Permission denied"}
+        return JsonResponse(ajax_response_data, status=status.HTTP_200_OK)
```

Comment:
@pkdash status code of 200 is not right here, 403 should be returned instead, right?